src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -112,14 +112,14 @@ private void createSymbolicLink() {
 
       // Try to delete the symbolic link in case a dangling link is present
       try {
-        Process process = Runtime.getRuntime().exec("unlink " + hdfsBinaryPath);
+        Process process = new ProcessBuilder("unlink", hdfsBinaryPath).start();
         redirectProcess(process);
         int exitCode = process.waitFor();
         if (exitCode != 0) {
           log.info("Unable to unlink old sym link. Link may not exist. Exit code: " + exitCode);
         }
       } catch (IOException e) {
-        log.fatal("Could not unlink" + hdfsBinaryPath + ": " + e);
```

Comment:
This should still be log fatal as before since we will exit if an error is thrown here.